doc/examples/perception_pipeline/CMakeLists.txt

```diff
@@ -1,13 +1,17 @@
-add_executable(cylinder_segment src/cylinder_segment.cpp)
-target_link_libraries(cylinder_segment ${catkin_LIBRARIES})
+install(DIRECTORY urdf
+  DESTINATION share/${PROJECT_NAME})
 
-add_executable(bag_publisher_maintain_time src/bag_publisher_maintain_time.cpp)
-target_link_libraries(bag_publisher_maintain_time ${catkin_LIBRARIES} ${Boost_LIBRARIES})
+install(DIRECTORY launch
+  DESTINATION share/${PROJECT_NAME})
 
-install(
-TARGETS
-    bag_publisher_maintain_time
-    cylinder_segment
-DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+install(DIRECTORY meshes
+  DESTINATION share/${PROJECT_NAME})
 
-install(DIRECTORY launch DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(DIRECTORY config
```

Comment:
You can merge the install commands like this:

```
install(DIRECTORY urdf worlds meshes ...
  DESTINATION share/${PROJECT_NAME})
```
The ... need to be replaced with the missing directory names

Reply:
Done it